### PR TITLE
Fix for 'requires accessible name'

### PR DESCRIPTION
**Pull request by sussexrick**
This adds a new property, `ariaLabelledBy`, which assigns an accessible name to the list with `role="listbox"`, enabling the first error identified in [#361](https://github.com/alphagov/accessible-autocomplete/issues/361) to be addressed.

It also removes the quotes around the build command in package.json, which enables the build command to run on Windows.

### CODE REVIEW

File: src/autocomplete.js [hunk 1]
```diff
@@ -50,7 +50,8 @@ export default class Autocomplete extends Component {
     required: false,
     tNoResults: () => 'No results found',
     tAssistiveHint: () => 'When autocomplete results are available use up and down arrows to review and enter to select.  Touch device users, explore by touch or with swipe gestures.',
-    dropdownArrow: DropdownArrowDown
+    dropdownArrow: DropdownArrowDown,
+    ariaLabelledBy: undefined | ''
```
Comment (JakeChampion) [inline src/autocomplete.js:54]: Is this meant to be an OR (`||`)?

Comment (sussexrick) [reply src/autocomplete.js]: @JakeChampion Yes, you're right. I've updated it.

File: src/autocomplete.js [hunk 2]
```diff
@@ -517,13 +519,14 @@ export default class Autocomplete extends Component {
           role='combobox'
           required={required}
           value={query}
-        />
+          />
```
Comment (JakeChampion) [inline src/autocomplete.js:522]:
```suggestion
        />
```

**Comment by JakeChampion** [comment]
Thanks @sussexrick, I'll look to release this to npm today

**Comment by sussexrick** [comment]
@JakeChampion Excellent, thank you. I'll create an identical PR for the original repo. We can hope.

**Comment by JakeChampion** [comment]
`@financial-times/accessible-autocomplete@2.2.0` is now on npm

**Pull request merged**